packages/devtools/src/devtoolsdriver.js

```diff
@@ -59,7 +59,7 @@ export default class DevToolsDriver {
          * check if command is implemented
          */
         if (typeof this.commands[command] !== 'function') {
-            return () => { throw new Error('Not yet implemented') }
+            return () => { throw new Error(`Command "${command}" is not yet implemented`) }
```

Comment:
I think you have to use `has` instead of `is` in present Perfect

Reply:
I think it is fine

Reply:
What if it is present simple?